src/OpenTK/Platform/MacOS/CocoaContext.cs

```diff
@@ -361,7 +356,15 @@ protected override void Dispose(bool disposing)
             IsDisposed = true;
         }
 
-        public override IntPtr GetAddress(IntPtr function)
+        // As of MacOS Monterey Beta 1/2, NSLookupSymbolInImage() stopped working. It is unclear
+        // if this will be fixed by Apple in the future. We therefore switch to dlsym, which is
+        // ever so slightly slower. We are keeping this code in case we ever want to switch back.
+        // A Feedback Assistant issue was open with Apple about this FB9319518.
+        //
+        // More information here: 
+        // https://github.com/opentk/opentk/issues/1308
+
+        public static IntPtr GetAddressUsingNSLookup(IntPtr function)
```

Comment:
I think that we can actually remove this function, as apple has said that NSLookup is deprecated and the above comment links to this issue, so people can find the old implementation through that.

Reply:
Ok, ill remove it one night this week.

-Mat

Reply:
Done! Left a comment, because why not.